Adyen/Core/Components/AbstractPersonalInformationComponent/NameFormItemInjector.swift

```diff
@@ -26,6 +29,7 @@ internal final class NameFormItemInjector: FormItemInjector, Localizable {
     /// :nodoc:
     internal lazy var item: FormTextInputItem = {
         let item = FormTextInputItem(style: style)
+        item.value = value ?? ""
```

Comment:
Now, `NameFormItemInjector` holds a value property (optional type) to enable the name's prefilling. Since the item's `value` is a non-optional type, I need to provide a fallback value if the prefilled information doesn't contain a first name.

It's the same default value we're already using in the parent class `FormTextItem`:
```swift
public init(style: FormTextItemStyle) {
    super.init(value: "", style: style)
}
```